devenv/docker/blocks/postgres/docker-compose.yaml

```diff
@@ -7,10 +7,18 @@
     ports:
       - "5432:5432"
     command: postgres -c log_connections=on -c logging_collector=on -c log_destination=stderr -c log_directory=/var/log/postgresql
+    healthcheck:
+      test: [ "CMD", "pg_isready", "-q", "-p", "5432",  "-d", "$$POSTGRES_DATABASE", "-U", "$$POSTGRES_USER" ]
```

Comment:
Think port can be skipped since that's the default.  What does the `-q` do? Just curious since we're using [this](https://github.com/grafana/grafana/blob/master/devenv/docker/ha_test/docker-compose.yaml#L52-L54) for ha_test setup

Reply:
```
-q
--quiet
Do not display status message. This is useful when scripting.
```
I think it does not harm to keep it.

Reply:
A good point :+1: